src/helpers/eventBridge.ts

```diff
@@ -162,6 +168,30 @@ export default class EventBridge {
     return result;
   }
 
+  async getAllEvents(): Promise<EventBridgeEvents> {
+    let allEventsFound = false;
+    const allEventBridgeEvents: EventBridgeEvents = {};
+
+    while (!allEventsFound) {
+      try {
+        const lastEventBridgeEvents = await this.getEvent();
+
+        if (!lastEventBridgeEvents || !lastEventBridgeEvents.Messages) {
+          return allEventBridgeEvents;
+        }
+
+        allEventBridgeEvents.Messages = [
+          ...(allEventBridgeEvents.Messages ?? []),
+          ...lastEventBridgeEvents.Messages,
+        ];
```

Comment:
I find this part a bit tricky to read. 

How about:

```ts
// initialise allEventBridgeEvents at the start of the function
const allEventBridgeEvents: EventBridgeEvents = { Messages: [] }

// in the while loop when we have a new event
allEventBridgeEvents.Messages.push(...lastEventBridgeEvent.Messages)
```

Would this work and bit a bit easier to read?